dss/api/bundles/__init__.py

```diff
@@ -130,13 +130,13 @@ def get(
     if link is None:
         response = make_response(jsonify(response_body), requests.codes.ok)
         response.headers['X-OpenAPI-Pagination'] = 'false'
-        return response
     else:
         response = make_response(jsonify(response_body), requests.codes.partial)
         response.headers['X-OpenAPI-Pagination'] = 'true'
-        response.headers['X-OpenAPI-Paginated-Content-Key'] = 'files'
         response.headers['Link'] = link
-        return response
+
+    response.headers['X-OpenAPI-Paginated-Content-Key'] = 'bundle.files'
```

Comment:
This change could might cause a test failure here:

https://github.com/HumanCellAtlas/data-store/blob/8be83e45114ffbd090966463650c83639d8bde9f/tests/test_bundle.py#L175

If it does, the failure would only be picked up after merge, because those tests are mistakenly marked as integration tests (will be fixed in #2495)

Reply:
I've added the required change to #2495